initializers/active_model_serializer.rb

```diff
@@ -0,0 +1 @@
+ActiveModel::ArraySerializer.root = false
```

Comment:
@jmontleon, I'm confused why if this isn't defined as `true` here rather than in the root_array_serializer